doc/md/guides/deploying/init.md

```diff
@@ -0,0 +1,160 @@
+---
+id: k8s-init-containers
+slug: /guides/deploying/k8s-init-container
+title: Deploying schema migrations to Kubernetes with Init Containers
+---
+
+In [Kubernetes](https://kubernetes.io), [Init Containers](https://kubernetes.io/docs/concepts/workloads/pods/init-containers/),
+are specialized containers that run before app containers in a Pod. Init containers
+can contain utilities or setup scripts not present in an app image. 
+
+Init containers can be utilized to run schema migrations with Atlas before the
+application loads. Because init containers can use a container image different
+from the application, developers can use a [purpose-built image](image.md) that
+only contains Atlas and the migrations scripts to run them.  This way, less 
+can be included in the application runtime environment, which reduces
+the attack surface from a security perspective. 
+
+Depending on an application's [deployment strategy](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#strategy),
+multiple replicas of an init container may run concurrently.  In the case of
+schema migrations, this can cause a dangerous race condition with unknown outcomes.
+To prevent this, in databases that support advisory locking, Atlas will acquire
+a lock on the migration operation before running migrations which makes the
+operation mutually exclusive.
+
+In this guide, we demonstrate how schema migrations can be integrated into
+a Kubernetes deployment using an init container.
+
+Prerequisites to the guide:
+1. [A migrations docker image](/guides/deploying/image)
+2. [A Kubernetes Deployment manifest](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/) defining
+   your application.
+3. A running Kubernetes cluster to work against.
+
+## Adding an init container
+
+Suppose our deployment manifest looks similar to this:
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nginx-deployment
+  labels:
+    app: nginx
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: nginx
+  template:
+    metadata:
+      labels:
+        app: nginx
+    spec:
+      containers:
+      - name: nginx
+        image: nginx:1.14.2
+        ports:
+        - containerPort: 80
+```
+
+Now, let's say our [migration container image](image.md) which contains the Atlas binary
+and our migration scripts is available at `ghcr.io/repo/migrations:v0.1.2`. We would like
```

Comment:
```suggestion
and our migration scripts are available at `ghcr.io/repo/migrations:v0.1.2`. We would like
```

Reply:
the migrations _are_, but the container _is_